pygmt/clib/session.py

```diff
@@ -1841,6 +1842,16 @@ def virtualfile_to_dataset(
         ...                 assert result is None
         ...                 assert Path(outtmp.name).stat().st_size > 0
         ...
+        ...     # strings output
```

Comment:
Maybe I should move all the doctests except the pandas one into the separate `test_clib_virtualfile.py` file since the docstrings are too long.